be/src/http/action/stream_load.cpp

```diff
@@ -301,6 +301,14 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, std::shared_ptr<Strea
         }
     }
 
+    if (ctx->body_bytes == 0 && !ctx->is_chunked_transfer) {
```

Comment:
There may be several things to consider here:
1. Forgot to set content length or chunked
2. Incorrect 0 set
3. It is itself 0